packages/nouns-contracts/contracts/governance/NounsDAOLogicV2.sol

```diff
@@ -901,6 +964,14 @@ contract NounsDAOLogicV2 is NounsDAOStorageV2, NounsDAOEventsV2 {
         }
     }
 
+    function _refundGas(uint256 startGas) internal {
```

Comment:
@eladmallel maybe we can do an early exit here in case the contract's balance is 0?
no reason to waste user's gas on this

Reply:
that's exactly what I was thinking -- there will be balance most of the time, so I didn't want to waste gas on checking, that's why it's implemented as it is. ok to keep it this way?

Reply:
yeah

Reply:
agreed, fixed https://github.com/nounsDAO/nouns-monorepo/pull/515/commits/69350411bfe1d6b34850ccf287e34a1881f1364c